src/main/java/hu/rezidencia/service/impl/CustomerServiceImpl.java

```
@@ -0,0 +1,93 @@
+            throw new IllegalArgumentException("Customer's name is not valid!");
+        } else if (customer.getAccountNumber() == null || !isValidCustomerAccountNumber(customer.getAccountNumber().trim())) {
+            throw new IllegalArgumentException("Customer's account number is not valid!");
+        } else {
+            return true;
+        }
+    }
+
+    private boolean isValidCustomerAccountNumber(String accountNumber) {
+        Pattern pattern = Pattern.compile(COSTUMER_ACCOUNT_NUMBER_PATTERN);
+        Matcher matcher;
+        matcher = pattern.matcher(accountNumber);
+        return matcher.matches();
+    }
+
+    private boolean isValidCustomerName(String name) {
+        Pattern pattern = Pattern.compile(CUSTOMER_NAME_PATTERN);
+        Matcher matcher;
+        matcher = pattern.matcher(name);
+        return matcher.matches();
+    }
+
+    public int importCustomers(File[] file) {
+        String line = null;
+        int counter = 0;
+        if (file != null) {
+            for (int j = 0; j < file.length; j++) {
+                try (BufferedReader br = new BufferedReader(new FileReader(file[j]))) {
+                    while ((line = br.readLine()) != null) {
+                        String[] parts = line.split(",");
+                        if (isValidCustomerName(parts[0])) {
+                            if (isValidCustomerAccountNumber(parts[1])) {
+                                Customer customer = new Customer(parts[0], parts[1]);
+                                addCustomer(customer);
+                                counter++;
+                            } else {
+                                customers.clear();
+                            }
+                        } else {
+                            customers.clear();
+                        }
+                    }
+
+
+                } catch (IOException e) {
+                    e.printStackTrace();
```

Noooooooo, never put this in code you upload somewhere. Throw an exception instead or log a message. Because you don't have a logger configured, you can throw an exception.